src/components/combo/combo-item.ts

```diff
@@ -31,6 +31,9 @@ export default class IgcComboItemComponent extends LitElement {
   @property({ type: Boolean, reflect: true })
   public active = false;
 
+  @property({ attribute: 'hide-checkbox', type: Boolean, reflect: true })
```

Comment:
```suggestion
  @property({ attribute: false, type: Boolean })
```
Unless there is a styling need, I don't see a reason to have a reflecting attribute here